crates/torii/core/src/processors/store_update_member.rs

```diff
@@ -90,30 +91,12 @@ where
             "Store update member.",
         );
 
-        let values_start = MEMBER_INDEX + 1;
-        let values_end: usize =
-            values_start + event.data[values_start].to_usize().context("invalid usize")?;
-
-        // Skip the length to only get the values as they will be deserialized.
-        let mut values = event.data[values_start + 1..=values_end].to_vec();
-
-        let tag = naming::get_tag(&model.namespace, &model.name);
-
-        if !db.does_entity_exist(tag.clone(), entity_id).await? {
-            warn!(
-                target: LOG_TARGET,
-                tag,
-                entity_id = format!("{:#x}", entity_id),
-                "Entity not found, must be set before updating a member.",
-            );
-
-            return Ok(());
-        }
```

Comment:
@Larkooo, totally fine to remove this check if it doesn't impact the rest of Torii.

The only downside would be that if the user try to query something that has only been updated, only the query by `HashedKeys` will be available, or am I missing an other check?

Reply:
It will have the same behaviour as the StoreUpdateRecord, so basically yes the keys is gonna be NULLED so hashed keys will only be available but anyway someone should never do that. Also the update will be ignored because the StoreUpdates perform UPDATE SQL statements on the entity which means that if there wasnt an entity previously, the new values wont be set